prov/rxm/src/rxm.h

```diff
@@ -239,14 +238,49 @@ struct rxm_recv_entry {
 };
 DECLARE_FREESTACK(struct rxm_recv_entry, rxm_recv_fs);
 
+#define RXM_QUEUE_HDR 		\
+	size_t 		size;	\
+	ofi_atomic32_t 	used;
```

Comment:
Introducing a define to replace declaring 2 variables doesn't seem worth it.  These aren't even part of a structure, where generic code could use them.  Can the circular queue from fi_rbuf.h be used here?